portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,94 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Pamela Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <script src="https://use.fontawesome.com/eac174ca5f.js"></script>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<!-- Navbar -->
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item active">
+        <a class="nav-link" href="#content">Home <span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#projects">Projects</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Gallery</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+<!-- End of Navbar -->
+<!-- Page Content -->
+<div id="content">
+    <img src="img/profile-img.jpg" class="profile-img">
+    <div class="textContainer">
+      <h1>Pamela's Portfolio</h1>
+      <p>Hello! My name is Pamela Lozano, I'm from Monterrey, Mexico and I study computer sciene engineering at 
+          Tecnologico de Monterrey (ITESM). My hobbies are reading and running, andI love dogs and music.
+         <br> My favorite book is: <i>The Outliers by Malcolm Gladwell</i>
+      </p>
+      <p>Contact info: pamelalozano16@gmail.com</p>
     </div>
+</div>
+    <div class="myContainer" id="projects">
+        <h2>Projects</h2>
+        <div class="container">
+  <nav class="navbar navbar-expand-lg navbar-dark" id="proj0" onclick="projectToggle(this.id)">
```

Comment:
\<nav> here doesn't make sense semantically. I think this should just be \<div>

Reply:
Yeah, you are right, it can be misunderstood. It's done I changed them, thanks!